lib/coarse_op.cuh

```diff
@@ -685,6 +718,24 @@ namespace quda {
     }
 
     bool advanceTuneParam(TuneParam &param) const {
+
+      if (use_mma && (type == COMPUTE_UV || type == COMPUTE_VUV)) {
+        constexpr bool query_max = true;
+        int max;
+        if (type == COMPUTE_UV) {
+          max = mma::launch_compute_uv_kernel<from_coarse, query_max>(param, arg, 1, 0);
```

Comment:
Will this launch a tune internally?

Reply:
With `query_max = true` the function will just return an `int`, which shouldn't launch a tune internally.